webapp/lib/deploymentProvider/PublishProfileDeploymentProvider.js

```diff
@@ -93,6 +93,9 @@ class PublishProfileDeploymentProvider {
                 username: res.userName,
                 password: res.userPWD
             };
+            console.log(`::add-mask::${creds.username}`);
+            console.log(`::add-mask::${creds.password}`);
+            console.log(`::add-mask::${creds.uri}`);

```

Comment:
Note that this should call core.setSecret but we're 1/2 done with the impl on that (it calls down to add-mask which just rolled out to the runner).  Create an issue for the change when it's done.

Reply:
Added - #3